### PR TITLE
rolldice: add page

### PR DESCRIPTION
**Pull request by vappolinario**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by bl-ue — CHANGES_REQUESTED** [review]
Cool @vappolinario.

Will you please add a "More information" link?  If you can't find any good official link, you can use https://manned.org/rolldice.  Here's an example: [`pages/common/goimports.md`](https://github.com/tldr-pages/tldr/blob/06bcfd34f08f89bf7562a570320f25e0678622e7/pages/common/goimports.md)

**Comment by vappolinario** [comment]
@bl-ue I found the command's github repo but the man page you linked seems more complete, which should i use?

**Comment by marchersimon** [comment]
@vappolinario in this case I'd say we should prefer <manned.org/rolldice>.

File: pages/linux/rolldice.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# rolldice
+
+> Roll virtual dice.
+
+- Roll a single 20 sided die:
```
Comment (marchersimon) [inline pages/linux/rolldice.md:5]:
```suggestion
- Roll a single 20 sided dice:
```
As far as I know, `dice` is more common than `die`.

File: pages/linux/rolldice.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+`rolldice d20`
```
Comment (marchersimon) [inline pages/linux/rolldice.md:7]:
```suggestion
`rolldice d{{20}}`
```
We keep any variables in `{{ }}`

File: pages/linux/rolldice.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+- Roll two six sided dice and keep the highest roll:
+
+`rolldice 2d6s1`
```
Comment (marchersimon) [inline pages/linux/rolldice.md:11]:
```suggestion
`rolldice {{2}}d{{6}}s{{1}}`
```

File: pages/linux/rolldice.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+- Roll two 20 sided dice and add a modifier value:
+
+`rolldice 2d20+5`
```
Comment (marchersimon) [inline pages/linux/rolldice.md:15]:
```suggestion
`rolldice {{2}}d{{20}}{{+5}}`
```

File: pages/linux/rolldice.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# rolldice
+
+> Roll virtual dice.
+
+- Roll a single 20 sided die:
+
+`rolldice d20`
+
+- Roll two six sided dice and keep the highest roll:
```
Comment (marchersimon) [inline pages/linux/rolldice.md:9]:
```suggestion
- Roll two six sided dice and drop the lowest roll:
```
I think that explains better what `s1` does.

**Review by marchersimon — CHANGES_REQUESTED** [review]

**Comment by bl-ue** [comment]
>@bl-ue I found the command's github repo but the man page you linked seems more complete, which should i use?
>
>https://github.com/sstrickl/rolldice

Personally, I like the GitHub repository because the first thing I want to know when I see a new command is the source/homepage of it.  But since this is old tool circa 1999 I think the man page is fine.

File: pages/linux/rolldice.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+- Roll two six sided dice and drop the lowest roll:
+
+`rolldice {{2}}d{{6}}s{{1}}`
+
+- Roll two 20 sided dice and add a modifier value:
+
+`rolldice {{2}}d{{20}}{{+5}}`
+
+- Roll a d20 two times:
```
Comment (marchersimon) [inline pages/linux/rolldice.md:18]:
```suggestion
- Roll a 20 sided dice two times:
```

**Review by marchersimon — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Looks fine to me! Thanks, @vappolinario :smiley_cat: 

In terms of the more info link, normally we would prefer to link to the GH repo, but in this case the GH repo has a very..... brief README, so the man page is preferable in this instance.

**Pull request merged**